src/Integration.UnitTests/LocalServices/ErrorListInfoBarControllerTests.cs

```diff
@@ -272,15 +274,15 @@ public void ErrorListInfoBarController_CurrentBackgroundProcessorCancellation()
             this.configProvider.ModeToReturn = SonarLintMode.LegacyConnected;
 
             // Act
-            testSubject.ProcessSolutionBinding();
+            await testSubject.ProcessSolutionBindingAsync();
 
             // Assert
             testSubject.CurrentBackgroundProcessor?.BackgroundTask.Should().NotBeNull("Background task is expected");
             testSubject.CurrentBackgroundProcessor.BackgroundTask.Wait(TimeSpan.FromSeconds(2)).Should().BeTrue("Timeout waiting for the background task");
             this.infoBarManager.AssertHasNoAttachedInfoBar(ErrorListInfoBarController.ErrorListToolWindowGuid);
 
-            // Act (refresh again and  let the blocked UI thread run to completion)
-            testSubject.ProcessSolutionBinding();
+            // Act (refresh again and let the blocked UI thread run to completion)
+            await testSubject.ProcessSolutionBindingAsync();
```

Comment:
There is still a synchronous `ProcessSolutionBinding` method. However, it calls `ProcessSolutionBindingAsync().Forget`, which returns immediately. So the tests now need to call the async method and wait for it to finish.